interface/modules/custom_modules/oe-module-dorn/public/results.php

```diff
@@ -0,0 +1,155 @@
+        let scriptTitle = 'get_lab_results.php?resultGuid=' + resultGuid +'&csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+        dlgopen(scriptTitle, '_blank', 800, 750, false, addTitle);
+    }
+    function ackResults(resultGuid, isRejected) {
+        top.restoreSession();
+        var addTitle = '<i class="fa fa-plus" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?>;
+        let scriptTitle = 'ack_lab_results.php?resultGuid=' + resultGuid +'&rejectResults='+ isRejected +'&csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+        dlgopen(scriptTitle, '_blank', 800, 750, false, addTitle);
+    }
+</script>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                require '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <div class="card">
+            <div class="card-body">
+                <h5 class="card-title"><?php echo xlt("DORN - Lab Pending Results"); ?></h5>
+                <div class="row">
+                    <div class="col">
+                        <form method="post" action="results.php">
+                            <div class="row">
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_labAcctNumber"><?php echo xlt("Lab Account Number") ?>:</label>
+                                            <input type="text" class="form-control" id="form_labAcctNumber" name="form_labAcctNumber" value="<?php echo isset($_POST['form_labAcctNumber']) ? attr($_POST['form_labAcctNumber']) : '' ?>"/>
+                                        </div>     
+                                    </div>
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_startDateTime"><?php echo xlt("Start Date") ?>:</label>
+                                            <input type="date" class="form-control" id="form_startDateTime" name="form_startDateTime" value="<?php echo isset($_POST['form_startDateTime']) ? attr($_POST['form_startDateTime']) : '' ?>"/>
+                                        </div>
+                                    </div>
+
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_endDateTime"><?php echo xlt("End Date") ?>:</label>
+                                            <input type="date" class="form-control" id="form_endDateTime" name="form_endDateTime" value="<?php echo isset($_POST['form_endDateTime']) ? attr($_POST['form_endDateTime']) : '' ?>"/>
```

Comment:
for date inputs recommend using the standard date widgets (see code elsewhere for examples)